loadtest/control/actions.go

```diff
@@ -249,14 +249,16 @@ func CreateAckPost(u user.User) UserActionResponse {
 		return UserActionResponse{Err: NewUserError(err)}
 	}
 
+	var urgent string = model.PostPriorityUrgent
+	var ack bool = true
 	postId, err := u.CreatePost(&model.Post{
 		Message:   "Priority Post Lorem ipsum dolor sit amet, consectetur adipiscing elit",
 		ChannelId: channel.Id,
 		CreateAt:  time.Now().UnixMilli(),
 		Metadata: &model.PostMetadata{
 			Priority: &model.PostPriority{
-				Priority:     model.NewString(model.PostPriorityUrgent),
-				RequestedAck: model.NewBool(true),
+				Priority:     &urgent,
+				RequestedAck: &ack,
```

Comment:
Can we follow the same style here and specify the values directly instead of defining them beforehand?
```suggestion
				Priority:     model.NewPointer(model.PostPriorityUrgent),
				RequestedAck: model.NewPointer(ack),
```